clang/test/CIR/CodeGen/complex-arithmetic.c

```diff
@@ -0,0 +1,128 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir -emit-cir -o %t.cir %s
+// RUN: FileCheck --input-file=%t.cir --check-prefixes=C,CHECK %s
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -x c++ -fclangir -emit-cir -o %t.cir %s
+// RUN: FileCheck --input-file=%t.cir --check-prefixes=CPP,CHECK %s
+
+double _Complex cd1, cd2;
+int _Complex ci1, ci2;
+
```

Comment:
Seems like we're missing tests for the unary ops?

Reply:
The tests for unary ops are here: https://github.com/llvm/clangir/pull/730/files/6261dc6010e5cb46cb9f780d7acf11442c8d8554#diff-af00df30674a0105a6f183bf20b1f389d2d586c6552a6a03a04c7f9ea5de41beR97-R128